app/screens/WishList/index.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
avoid this by using a `PureComponent` instead of `Component`. The import is the same as for the component